Source/CloudyPanelPlugin/Private/CloudyPanelPlugin.cpp

```diff
@@ -13,41 +17,20 @@ DEFINE_LOG_CATEGORY(ModuleLog)
 #define IP "127.0.0.1"
 #define PORT_NO 55555
```

Comment:
I believe you are not suppose to use port `55555` during our last discussion, because I have mentioned that this port has been used by the RemoteController plugin. But you can still use any of the our reserved port between `55555-55559`. 

I think there is some misunderstanding in the [docs](https://docs.google.com/document/d/1n0v-JUChb7avvgTZyGbsM2SGJ10Emb2A3uxnGZ75BAc/edit). Because CloudyPanel doesn't need 10 ports.